src/main/flight/pid.h

```diff
@@ -113,6 +113,13 @@ PG_DECLARE_ARRAY(pidProfile_t, MAX_PROFILE_COUNT, pidProfiles);
 
 typedef struct pidConfig_s {
     uint8_t pid_process_denom;              // Processing denominator for PID controller vs gyro sampling rate
+    uint8_t runaway_takeoff_prevention;          // off, on - enables pidsum runaway disarm logic
+    uint8_t runaway_takeoff_threshold;           // runaway pidsum trigger threshold
+    uint8_t runaway_takeoff_activate_delay;      // delay in ms where pidSum is above threshold before activation
+    uint8_t runaway_takeoff_deactivate_throttle; // minimum throttle percent required during deactivation phase
+    uint8_t runaway_takeoff_deactivate_pidlimit; // pidSum limit during deactivation phase
+    uint8_t runaway_takeoff_deactivate_stick;    // minimum stick deflection during deactivation phase
+    uint16_t runaway_takeoff_deactivate_delay;   // delay in ms for "in-flight" conditions before deactivation (successful flight)
```

Comment:
Move this up to below `runaway_takeoff_activate_delay` to prevent padding.

Reply:
Done.